geemap/toolbar.py

```diff
@@ -163,7 +179,7 @@ def _reset_others(self, current):
             if other is not current:
                 other.value = False
 
-    def _toggle_callback(self, m, selected):
+    def _toggle_callback(self, m, selected, _):
```

Comment:
Nit: `_toggle_callback(self, m, selected, item)` and then call `del item  # Unused.`.

Reply:
Done.